arlo-client/src/components/Audit/index.tsx

```diff
@@ -100,7 +112,9 @@ const Audit: React.FC<IProps> = ({
         updateAudit={updateAudit}
       />
 
-      {isAuthenticated ? (
+      {isAuthenticated &&
+      /* istanbul ignore next */
```

Comment:
Why?

Reply:
@eventualbuddha Because both branches had the same functionality so far, and it didn't matter for this iteration, and I couldn't get the mock to send both values. I tackled it again this morning, though and got it working, so this can be removed. :slightly_smiling_face: